src/__experimental__/components/simple-color-picker/simple-color-picker.spec.js

```diff
@@ -16,6 +16,32 @@ const colorValues = [
 ];
 const name = 'test-group';
 
+function renderAttached(renderer = mount, props, childProps) {
```

Comment:
Again I think you could get away with just adding `{ attachTo: document.getElementById('enzymeContainer') }` to the `render` function